src/subclass/mod.rs

```diff
@@ -9,10 +9,10 @@ pub mod output_stream;
 pub mod seekable;
 
 pub mod prelude {
-    pub use super::application::{ApplicationImpl, ArgumentList};
-    pub use super::input_stream::InputStreamImpl;
-    pub use super::io_stream::IOStreamImpl;
-    pub use super::output_stream::OutputStreamImpl;
+    pub use super::application::{ApplicationImpl, ApplicationImplExt, ArgumentList};
```

Comment:
I wonder why `ArgumentList` is here. Consistent with other things it should be directly in `mod.rs` and not in the prelude as it's not a trait.

Reply:
`pub use application::ArgumentList;` outside `mod prelude` or only remove it from prelude ?
There are no other structs in subclass/*

Reply:
I'd move it from the prelude to the subclass module

Reply:
Same question about glib, it has `Boxed`. `Property`, `InstanceStruct`, `ClassStruct`, `InitializingType`, `TypeData`, `SignalInvocationHint`, `SignalClassHandlerToken`.

Reply:
I not sure that import them by full path is good idea.
IMHO better reexport them and prelude, hide submodules and use as `glib::subclass::Boxed` etc.

Reply:
Sounds good to me

Reply:
Done